code/modules/jobs/map_changes/map_changes.dm

```diff
@@ -1,13 +1,4 @@
 //this needs to come after the job_types subfolder to keep the correct ordering
 
-<<<<<<< HEAD
-#include "..\..\..\..\_maps\map_files\Deltastation\job_changes.dm"
-#undef JOB_MODIFICATION_MAP_NAME
-#include "..\..\..\..\_maps\map_files\MetaStation\job_changes.dm"
-#undef JOB_MODIFICATION_MAP_NAME
-#include "..\..\..\..\_maps\map_files\OmegaStation\job_changes.dm"

```

Comment:
put these back at once, they're [quite necessary](https://github.com/yogstation13/Yogstation-TG/commit/ba239f001442d2e53dd40223bffb0cb47654ae3e#diff-b20643f29a86b651389b562fb62f2b86)

Reply:
just not the omegastation one